wordle-web/components/LeaderBoard2.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-container>
+    <v-btn @click="refresh">Refresh</v-btn>
+
+    <v-container v-if="!hasData">
+      There are no leaderboard entries at this time...
+    </v-container>
+
+    <v-container>
+      <v-row justify="center">
+        <v-col cols="10">
+          <v-card color="transparent" flat>
+            <v-card-title class="justify-center"> </v-card-title>
+            <v-card color="transparent" flat>
+              <v-card-title class="justify-center text-h3 font-weight-bold">
+                <starLogo /> <starLogo /> <starLogo /> Top Awesome Players
+                <starLogo /> <starLogo /> <starLogo />
+              </v-card-title>
+            </v-card>
+          </v-card>
+        </v-col>
+      </v-row>
+    </v-container>
+
+    <v-container>
+      <v-row>
+        <v-col col="3" justify="center" class="text-h5">Name</v-col>
+        <v-col col="3" justify="center" class="text-h5">Game Count</v-col>
+        <v-col col="3" justify="center" class="text-h5">Average Guesses</v-col>
+        <v-col col="3" justify="center" class="text-h5"
+          >Average Second/Game</v-col
+        >
+      </v-row>
+    </v-container>
+
+    <v-container v-if="hasData" class="item">
+      <v-row
+        v-for="t in arr"
+        :key="t"
+        justify="center"
+        color="green"
+        class="text-h6"
+      >
+        <v-col col="3" justify="center">
+          {{ t.name }}
+        </v-col>
+
+        <v-col cols="3" justify="center">
+          {{ t.gameCount }}
+        </v-col>
+
+        <v-col cols="3" justify="center">
+          {{ t.averageGuesses }}
+        </v-col>
+
+        <v-col cols="3" justify="center">
+          {{ t.averageSecondsPerGame }}
+        </v-col>
+      </v-row>
+    </v-container>
+  </v-container>
+
+  <!-- <v-container v-if="hasData">
+    <v-row 
+      v-for="t in arr"
+      :key="t"
+      justify="center"
+      >
+      <v-col col="5"  justify="center">
+        {{t.name}}
+      </v-col>
+
+      <v-col cols="2">
+        {{t.gameCount}}
+      </v-col>
+
+      <v-col cols="2">
+        {{t.averageGuesses}}
+      </v-col>
+      <v-col cols="3">
+        {{t.averageSecondsPerGame}}
+      </v-col>
+    </v-row>
+    </v-container> -->
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import Component from 'vue-class-component'
+
+@Component
+export default class LeaderBoard2 extends Vue {
+  arr: any[] = []
+  hasData = false
+
+  beforeMount() {
```

Comment:
```suggestion
  created() {
```
This will work better for you